### PR TITLE
fix(replace): change from standard to multi loop characteristics

### LINKED ISSUE #1673: Multi-instance toggle loop then toggle Parallel or Sequential XML label No updates
__Describe the Bug__

GIF 2022-6-7 19-50-50

__Steps to Reproduce__

1. Create Task
2. Add loop multi-instance
3. toggle Parallel or Sequential

__Expected Behavior__

XML  label： <bpmn:multiInstanceLoopCharacteristics /> or  <bpmn:multiInstanceLoopCharacteristics isSequential="true" />

__Environment__

 - Library version: 9.1.0

### PR DESCRIPTION
**Pull request by smbea**
Closes #1673

The problem was that it was always reusing the same loopCharacteristics (in order to maintain the properties), and the change between multi instance and standard loopCharacteristics didn't occur.

I added test cases to ensure the changes between them aren't broken.

### CODE REVIEW

File: lib/features/popup-menu/ReplaceMenuProvider.js [hunk 1]
```diff
@@ -427,18 +427,29 @@ ReplaceMenuProvider.prototype._getLoopEntries = function(element) {
   var translate = this._translate;
 
   function toggleLoopEntry(event, entry) {
-    var loopCharacteristics = getBusinessObject(element).loopCharacteristics;
+    const {
```
Comment (nikku) [inline lib/features/popup-menu/ReplaceMenuProvider.js:430]: We should use ES5 (`var`) here.

Comment (nikku) [reply lib/features/popup-menu/ReplaceMenuProvider.js:430]: Also, we should not use destructuring.

You can verify things work locally (ES5-style) via 

```
TEST_BROWSERS=PhantomJS npm test
```

File: test/spec/features/popup-menu/ReplaceMenuProviderSpec.js [hunk 2]
```diff
@@ -511,6 +511,24 @@ describe('features/popup-menu - replace menu provider', function() {
...
+
+        openPopup(task);
+
+        // when
+        triggerAction('toggle-parallel-mi');
+
+        // then
+        var newLoopCharacteristics = businessObject.loopCharacteristics;
+
+        expect(is(newLoopCharacteristics, 'bpmn:MultiInstanceLoopCharacteristics')).to.be.true;
```
Comment (nikku) [inline test/spec/features/popup-menu/ReplaceMenuProviderSpec.js:528]: Should we also verify the value of `isSequential`?

File: test/spec/features/popup-menu/ReplaceMenuProviderSpec.js [hunk 3]
```diff
@@ -617,6 +635,24 @@ describe('features/popup-menu - replace menu provider', function() {
...
+
+        openPopup(task);
+
+        // when
+        triggerAction('toggle-sequential-mi');
+
+        // then
+        var newLoopCharacteristics = businessObject.loopCharacteristics;
+
+        expect(is(newLoopCharacteristics, 'bpmn:MultiInstanceLoopCharacteristics')).to.be.true;
```
Comment (nikku) [inline test/spec/features/popup-menu/ReplaceMenuProviderSpec.js:652]: Should we also verify the value of `isSequential`?

File: test/spec/features/popup-menu/ReplaceMenuProviderSpec.js [hunk 4]
```diff
@@ -720,6 +756,24 @@ describe('features/popup-menu - replace menu provider', function() {
...
+
+        openPopup(task);
+
+        // when
+        triggerAction('toggle-loop');
+
+        // then
+        var newLoopCharacteristics = businessObject.loopCharacteristics;
+
+        expect(is(newLoopCharacteristics, 'bpmn:StandardLoopCharacteristics')).to.be.true;
```
Comment (nikku) [inline test/spec/features/popup-menu/ReplaceMenuProviderSpec.js:775]: Should we also verify `isSequential` is not set?

File: test/spec/features/popup-menu/ReplaceMenuProviderSpec.js (hunk 2 above)
Comment (smbea) [reply test/spec/features/popup-menu/ReplaceMenuProviderSpec.js:528]: Although that wasn't broken, it doesn't hurt to add an extra layer to the tests

**Review by nikku — APPROVED** [review]

**Pull request merged**